test/testscript.py

```diff
@@ -0,0 +1,6 @@
+import pymongo
+import sys
+print(sys.argv)
+client = pymongo.MongoClient(serverSelectionTimeoutMS=1000)
```

Comment:
Let's continue to follow good conventions in the test script - leave 2 empty lines after the import and before the code begins.